manager/integration/tests/test_upgrade.py

```diff
@@ -162,23 +188,31 @@ def test_upgrade(longhorn_upgrade_type,
     host_id = get_self_host_id()
     pod_data_path = "/data/test"
 
-    pod_volume_name = generate_volume_name()
-
-    auto_salvage_setting = client.by_id_setting(SETTING_AUTO_SALVAGE)
-    setting = client.update(auto_salvage_setting, value="false")
-
-    assert setting.name == SETTING_AUTO_SALVAGE
-    assert setting.value == "false"
-
-    # Create Volume attached to a node.
-    volume1 = create_and_check_volume(client,
-                                      volume_name,
-                                      size=SIZE)
-    volume1.attach(hostId=host_id)
-    volume1 = wait_for_volume_healthy(client, volume_name)
-    volume1_data = write_volume_random_data(volume1)
+    # Disable Auto Salvage Setting
+    update_setting(client, SETTING_AUTO_SALVAGE, "false")
+
+    # 2-1 Create vol_revision_enabled with revision counter enabled
+    # attached to a node
+    update_setting(client, SETTING_DISABLE_REVISION_COUNTER, "true")
```

Comment:
This is for enabling revision counter, so the setting should be `false`. The same issue for disabling RC below.

Reply:
1. Add validation to check volume engine image has been upgraded
2. Correct revision counter enabled|disable configuration
      - https://github.com/longhorn/longhorn-tests/pull/1168/commits/8742647249b7319910237f6d83c2d4d233b94d76